spec/features/sidebar_spec.rb

```diff
@@ -8,4 +8,25 @@
 
     expect(active_link.text).to eq "Customers"
   end
+
+  it "displays translated name of model" do
+    translations = {
+      activerecord: {
+        models: {
+          customer: {
+            one: "User",
+            other: "Users",
+          },
+        },
+      },
+    }
+
+    with_translations(:en, translations) do
+      visit admin_customers_path
+
+      sidebar = find(".sidebar__list")
```

Comment:
We're using [BEM syntax][BEM] for our CSS. In this case, the CSS class would read as: "The list element inside the sidebar block", or something like that.

CSS selector formats are something that we could have a whole other discussion on - I'm still not sure what the best solution is for Administrate. I chose BEM because many designers at thoughtbot have had pleasant experiences working with it, and it makes it a bit easier to write maintainable (S)CSS.

[BEM]: http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/.